packages/cli/src/lib/snapshot/pullModel/interfaces.ts

```diff
@@ -1,31 +1,10 @@
-import {ResourceSnapshotType} from '@coveord/platform-client';
+import {ResourceSnapshotExportConfigurationModel} from '@coveord/platform-client';
 
-export type SnapshotPullModelResourceType =
```

Comment:
I don't think we should force a list of the resources to include in the SPM.
An advanced user could possibly want to include source mappings, QREs, thesaurus, ...